mod/gc_group_layout/views/default/groups/profile/summary.php

```diff
@@ -75,7 +75,7 @@
 
                 	} else {
 
-		                $group_name = ($group->title3) ? gc_explode_translation($group->title3, $lang) : $group->name;
+		                $group_name = ($group->name) ? gc_explode_translation($group->name, $lang) : $group->name;
```

Comment:
the false condition is unreachable in this case and ends up behaving like gc_explode_translation() anyways.